Sources/SAPCAI/UI/MessageView/QuickRepliesMessageView.swift

```diff
@@ -18,20 +23,26 @@ struct QuickRepliesMessageView: View {
         assertionFailure("only use `QuickRepliesMessageView` with message type `quickReplies`")
         return nil
     }
+
+    var amountOfButtons: Int {
+        let limitForVisibleButtons = 11
```

Comment:
So in this case we are limiting number of quick reply buttons to 11?

Reply:
@justinguo yes, I introduced this in a different commit to align behavior with WebClient

Reply:
@MarcoEidinger Got it! So the solution looks good to me.